src/queues.c

```diff
@@ -311,6 +310,21 @@ void queues_notification_close_id(int id, enum reason reason)
 {
         struct notification *target = NULL;
 
+        char* reason_string;
```

Comment:
This code is suited better somewhere else (depending on its exact purpose).

When we're here:

- We are saying we're closing a notification. (Which notification!?)
- We are not sure, whether there is even a notification, we're able to close. (`target` may be `NULL`)
- We are not sure, if we will even send a DBus signal `NotificationClosed`. (`target.dbus_valid` may be `false`)


Please put it either into `if (target)` or put it to `dbus.c` with the `signal_notification_closed` handling.

Reply:
This was just a quick hack for debugging purposes. I think the log might be useful later, so I'll move it to signal_notification_closed.